src/develop/ui/patterns/mobile/carousel.md

```diff
@@ -133,100 +126,86 @@ Here's what it would look like:
 
 #### Displaying a Button when Viewing the Last Carousel Item
 
-1\. Create a **boolean** local variable on the screen called **ShowButton**.
+1. Create a **boolean** local variable on the screen called **ShowButton**.
 
-2\. Create an action called **OnChange** and add it to the **OnItemChange**
+1. Create an action called **OnChange** and add it to the **OnItemChange**
 handler of the Carousel.
 
-3\. In this action, we will verify if the current position is equal to the
+1. In this action, we will verify if the current position is equal to the
 total number of elements in the OnBoarding’s Carousel.
 
 ![](images/Carousel_onboarding.png)
 
-4\. The logic is: if **“Index = 2”** the local variable is assigned to _True_
+1. The logic is: if **“Index = 2”** the local variable is assigned to _True_
 , otherwise it is assigned to _False_ .
+1. The variable **showButton** shows the button to the user.
 
-2\. The variable **showButton** shows the button to the user.
-
-3\. Drag the **FloatingContent** pattern (Content\FloatingContent) into the
+1. Drag the **FloatingContent** pattern (Content\FloatingContent) into the
 screen.
 
-7\. Add a **Container** and in the **Visible** parameter, set the local
+1. Add a **Container** and in the **Visible** parameter, set the local
 variable **showButton**.  
 a. Drag the “ **Animate** ” block into the container, and set the **Animation
 Type**.  
 b. Add a button into the Content placeholder.  
   
-8\. Publish your app.
+1. Publish your app.
 
-### Creating a Carousel with Buttons to Navigate
+### Creating a carousel with navigation buttons
 
 This pattern includes [public actions](<public-actions.md>) that you can use without having to use the Carousel's own
 navigation (Next, Previous or GoTo). In this use case, we will only use
 buttons to navigate in the Carousel:
 
-1\. Create an action and drag either one of the public actions Next, Previous
+1. Create an action and drag either one of the public actions Next, Previous
 or GoTo.
 
-2\. Create a button to associate the **onClick** event with the created
+1. Create a button to associate the **onClick** event with the created
 action.
 
-3\. See the image below with all possible actions.
+1. See the image below with all possible actions.
 
 ![](images/Carousel__on_click.png)
 
 ### Creating a ListRemove in Carousel
 
 This pattern includes the **UpdateCarousel** [public actions](<public-actions.md>) to update all variables and positions in the Pattern:
```

Comment:
What's the use case we're trying to describe here? In what scenario would it be valuable to use a ListRemove in a Carousel? "Creating a ListRemove in Carousel" is too obscure for it to be useful as it is. @pacferreira